pkg/pillar/go.sum

```diff
@@ -2968,6 +2976,8 @@ k8s.io/client-go v0.22.5/go.mod h1:cs6yf/61q2T1SdQL5Rdcjg9J1ElXSwbjSrW2vFImM4Y=
 k8s.io/client-go v0.26.2/go.mod h1:u5EjOuSyBa09yqqyY7m3abZeovO/7D/WehVVlZ2qcqU=
 k8s.io/client-go v0.29.0 h1:KmlDtFcrdUzOYrBhXHgKw5ycWzc3ryPX5mQe0SkG3y8=
 k8s.io/client-go v0.29.0/go.mod h1:yLkXH4HKMAywcrD82KMSmfYg2DlE8mepPR4JGSo5n38=
+k8s.io/client-go v0.29.3 h1:R/zaZbEAxqComZ9FHeQwOh3Y1ZUs7FaHKZdQtIc2WZg=
```

Comment:
Could you try to run `go mod tidy`? I wonder if the previous versions of client-go and other Kubernetes packages will go away from go.sum.

Reply:
bash-3.2$ go mod tidy
go: go.mod file indicates go 1.21, but maximum version supported by tidy is 1.20

Reply:
You should upgrade `go` on your laptop to 1.21 before running `go mod tidy`.

Reply:
I pushed new commit with "go mod tidy"